tasks/main.yml

```diff
@@ -86,13 +87,6 @@
     mode: 0644
   notify: restart amazon-cloudwatch-agent
 
-- name: Install Python modules
-  become: yes
-  pip:
-    name:
-      - boto3
-  delegate_to: localhost
-
 - name: Setup log groups
```

Comment:
Boto3 should be installed before running this role. I can't see any reason to make the role install it, instead if boto3 isn't installed then the role should fail.

Reply:
True I guess